stdlib/public/core/Integers.swift.gyb

```diff
@@ -2898,6 +2898,9 @@ public struct ${Self}
   : FixedWidthInteger, ${Unsigned}Integer,
     _ExpressibleByBuiltinIntegerLiteral {
 
+  public typealias IntegerLiteralType = ${Self}
```

Comment:
How is this related?

Reply:
Hum. It isn't. I probably meant to do this on my clean-up branch. But since this is a cherry-pick from master where it's merged, probably should just leave it in.